pkgs/games/flightgear/default.nix

```diff
@@ -6,8 +6,8 @@
 }:
 
 let
-  version = "2018.2.2";
-  shortVersion = "2018.2";
+  version = "2018.3.1";
+  shortVersion = "2018.3";
```

Comment:
Shouldn't there be a hash change below?

Reply:
Should have marked WIP, going through build steps, still need to push :smile: